_mods/the-enclave/metadata.json

```diff
@@ -1,11 +1,11 @@
 ---
 mod-id: the-enclave
 title: The Enclave
-version: "0.0.8b"
+version: "0.0.9"
 lastUpdated: 1474848000000
 author: Storm Crow
-description: "A new galaxy, way unbalanced outfits, and a rubbish story.
-  Fun times."
+description: "During the Alpha Wars, a small fleet of ships disappeared from human space.
+Five hundred years later, a wormhole to another galaxy opens."
```

Comment:
Something is up with the format here. It might need two spaces before "Five". Note the space in front of "Fun times." in the previous file. 

https://makuta-miras.github.io/ES-Mod-Share/
Here, you'll see what it'll look like if this is merged.